File: Bronze_python/1934.py
# 최소공배수

import math

def lcm(x, y):
  return x * y // math.gcd(x, y)

num = int(input())

for i in range(num):
  num1, num2 = map(int, input().split(' '))
  print(lcm(num1, num2))

File: Bronze_JavaScript/1934.js
// 최소공배수

const fs = require("fs");

const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const gcd = (a, b) => {
  while (b > 0) {
    let temp = b;
    b = a % b;
    a = temp;
  }

  return a;
};

const lcm = (a, b) => {
  return (a * b) / gcd(a, b);
};

const num = parseInt(input[0]);

for (let i = 1; i <= num; i++) {
  let [num1, num2] = input[i].split(" ").map(Number);

  console.log(lcm(num1, num2));
}
